xata/client.py

```diff
@@ -127,7 +127,7 @@ def __init__(
             "connection": "keep-alive",
             "x-xata-client-id": str(uuid.uuid4()),
             "x-xata-session-id": str(uuid.uuid4()),
-            "x-xata-agent": f"client=PY_SDK;version={__version__};",
+            "x-xata-agent": f"client=PY_SDK; version={__version__}",
```

Comment:
want to add a comment explaining the format?

thinking: the `x-xata-agent` field is not generated from a function. when the next person makes a change, we can help by making it clear what the format is + underline that the format remains consistent